module/localisation/BallLocalisation/src/BallLocalisation.cpp

```diff
@@ -57,7 +57,7 @@ namespace localisation {
 
         /* Run Time Update */
         on<Every<15, Per<std::chrono::seconds>>, Sync<BallLocalisation>, With<FieldDescription>, With<Sensors>>().then(
-            "BallLocalisation Time", [this](const FieldDescription& field, const Sensors& sensors) {
+            "BallLocalisation Time", [this]() {
```

Comment:
When you remove all the arguments you don't need the parenthesis (although I think this is a personal choice)
```cpp
"BallLocalisation Time", [this] {
```
is equally fine.

Reply:
You could also do 
`"BallLocalisation Time", [this](const FieldDescription& /*field*/, const Sensors& /*sensors*/)`

Not that it makes a lot of difference

Reply:
Actually, if we're not using them why don't we just remove the With<> parts?

Reply:
This is also a valid point